app/controllers/distributions_controller.rb

```diff
@@ -1,5 +1,12 @@
 class DistributionsController < ApplicationController
   skip_before_action :set_distributions
+  before_action :set_parameters
+
+  def set_parameters
+    @testing = true
```

Comment:
this variable is only used in app/views/distributions/index.html.erb, why setting it for all actions?
Even more, do you really needed it? you could use `@testing_version` in app/views/distributions/index.html.erb 🤔

Reply:
How do do that? I have no clue about rails :)

Reply:
Instead of `<% if @testing %>` you can write `@testing_version`.

As `@testing_version` is an object (String), using it in an `if` in Ruby would be the same as using `true`.